samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/ChatController.cs

```diff
@@ -138,7 +138,7 @@ public ChatController(ILogger<ChatController> logger, ITelemetryService telemetr
         {
             var chatId = ask.Variables.Where(v => v.Key == "chatId").First().Value;
             await messageRelayHubContext.Clients.Group(chatId).SendAsync(ReceiveResponseClientCall, chatSkillAskResult, chatId);
-            await messageRelayHubContext.Clients.Group(chatId).SendAsync(GeneratingResponseClientCall, chatId, false);
+            await messageRelayHubContext.Clients.Group(chatId).SendAsync(GeneratingResponseClientCall, chatId);
```

Comment:
nit: If this is going to send an empty string automatically, probably want to make that explicit.